pxt.json

```diff
@@ -3,7 +3,9 @@
     "version": "1.0.15",
     "description": "A iBit package for pxt-microbit",
     "license": "MIT",
-    "dependencies": {},
+    "dependencies": {
```

Comment:
[run] git status --porcelain --untracked-files=no
ERROR: Package not installed: core

How fix it?

Reply:
run ```pxt install```